services/site/src/components/homepage/TopChallengeSection.tsx

```diff
@@ -0,0 +1,24 @@
+import clsx from "clsx";
+import Link from "next/link";
+import React from "react";
+
+const TopChallengeSection: React.FunctionComponent<React.PropsWithChildren> = ({ children }) => {
+  return (
+    <section className={clsx("flex flex-col items-center my-12 py-12 px-4 lg:p-12 container-dark", "sm:my-20", "xl:my-24")}>
+      <h2 className={clsx("text-center h4 mb-10", "sm:h3", "xl:h2")}>Our three most popular challenges</h2>
```

Comment:
They aren't the most popular ones. I think we should change this to something more ambiguous like "Featured challenges" or "Loved by our community"